### PR TITLE
jonny class

### PR DESCRIPTION
**Pull request by jonny5891**
please review creation of jonny class code

### CODE REVIEW

File: src/develop/Jonny.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package develop;
+
+public class Jonny {
+    //created jonny
```
Comment (srestr53) [inline src/develop/Jonny.java:4]: Hey jonny create a jonny method , don't forget to make it static

**Comment by srestr53** [comment]
Hey Jonny , check your code  and make the changes needed

File: src/develop/Jonny.java (hunk 1 above)
Comment (Lokmanbzd) [inline src/develop/Jonny.java:4]: do you see my comment

**Review by srestr53 — APPROVED** [review]

File: src/develop/Jonny.java (hunk 1 above)
Comment (srestr53) [inline src/develop/Jonny.java:4]: I agree with Lokman , thx

**Review by srestr53 — APPROVED** [review]

File: src/develop/Jonny.java (hunk 1 above)
Comment (Lokmanbzd) [inline src/develop/Jonny.java:4]: do you see my comment

**Review by Lokmanbzd — APPROVED** [review]

File: src/develop/Jonny.java (hunk 1 above)
Comment (jonny5891) [reply src/develop/Jonny.java:4]: no thats not right

Comment (jonny5891) [reply src/develop/Jonny.java:4]: no

**Pull request merged**